src/Microsoft.AspNet.Mvc.Core/FileResult.cs

```diff
@@ -25,7 +25,7 @@ public abstract class FileResult : ActionResult
         /// </summary>
         /// <param name="contentType">The Content-Type header of the response.</param>
         protected FileResult(string contentType)
-            : this(new MediaTypeHeaderValue(contentType))
+            : this(MediaTypeHeaderValue.Parse(contentType))
```

Comment:
This constructor overload doesn't seem to be used except in tests. Can we remove it? If not, should test using the `EmptyFileResult`.